crates/noirc_evaluator/src/ssa/acir_gen.rs

```diff
@@ -1014,7 +982,7 @@ pub fn evaluate_zero_equality(x: &InternalVar, evaluator: &mut Evaluator) -> Wit
 }
 
 /// Creates a new witness and constrains it to be the inverse of x
-pub fn evaluate_inverse(x: &mut InternalVar, evaluator: &mut Evaluator) -> Witness {
+fn evaluate_inverse(mut x: InternalVar, evaluator: &mut Evaluator) -> Witness {
```

Comment:
does x need to be mutable?

Reply:
It was required by get_or_generate_witness, but looking at get_or_generate_witness that didn't need to be mutable so I changed both